docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+version: "3.7"
+
+services:
+  server:
+    image: shieldbattery/${SERVER_NAME}:${SERVER_VERSION:-latest}
+    container_name: sb-${SERVER_NAME}-server
+    # Required for proper handling of kernel signals (eg. ctrl-c); for more info, see:
+    # https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#handling-kernel-signals
+    init: true
+    ports:
+     - "80:5555"
+    volumes:
+      - uploaded_files:${UPLOADED_FILES_PATH:-./server/uploaded_files}
+      - type: bind
```

Comment:
This syntax is pretty much never used for bind volumes, and even the short version has a way to specify read only (although the bash evaluation makes it slightly less readable):

`- ${SERVER_CONFIG_PATH:-./server/config.js}:/home/node/shieldbattery/server/config.js:ro
- ${SERVER_DB_CONFIG_PATH:-./server/database.json}:/home/node/shieldbattery/server/config.js:ro`

Reply:
I think this was intended to not be a bind volume, I just misunderstood when I was initially using this. It is/was at least declared in `volumes` and is intended to be filled by the server when people upload things, so I think this usage is probably fine.